.gitignore

```diff
@@ -79,6 +79,7 @@ fabfile/
 .sass-cache/
 node_modules/
 bower_components/
+frontend/index.html
 
```

Comment:
@aka-jain : Should i add `frontend/dist/` here to make difference between frontend and backend??